forge-gui/res/cardsfolder/upcoming/dynaheir_invoker_adept.txt

```diff
@@ -0,0 +1,12 @@
+Name:Dynaheir, Invoker Adept
+ManaCost:1 U R W
+Types:Legendary Creature Human Wizard
+PT:4/4
+K:Haste
+S:Mode$ ActivateAbilityAsIfHaste | Affected$ Creature.Other+YouCtrl | Description$ You may activate abilities of other creatures you control as though those creatures had haste.
+A:AB$ Effect | Cost$ T | Triggers$ ActivateTrig | Duration$ Permanent | SpellDescription$ When you next activate an ability that isn't a mana ability this turn by spending four or more mana to activate it, copy that ability. You may choose new targets for the copy.
```

Comment:
No Duration$?

Reply:
I've got a `Duration$ Permanent` in there

Reply:
I meant why is it there?

The ability says "this turn"

Reply:
Also wrong TriggerDescription

Reply:
Gotcha! Thanks 😅